packages/fontless/src/vite.ts

```diff
@@ -47,7 +47,11 @@ export function fontless(_options?: FontlessOptions): Plugin {
 
       cssTransformOptions = {
         processCSSVariables: options.processCSSVariables,
-        shouldPreload: () => false,
+        shouldPreload(fontFamily, _fontFace) {
+          const override = options.families?.find(f => f.name === fontFamily)
+          const merged = { ...options.defaults, ...override }
+          return merged.preload ?? false
```

Comment:
what about (to avoid the admittedly minimal!) cost of instantiating a new object?

```suggestion
          return override?.preload ?? options.defaults.preload ?? false
```